.env

```diff
@@ -41,6 +41,14 @@ VPN_CLIENT_NAME=default-management-vpn
 DEBUG_MODE=False
 DJANGO_LOG_LEVEL=INFO
 USE_OPENWISP_CELERY_TASK_ROUTES_DEFAULTS=True
+OPENWISP_CELERY_CONCURRENCY=1
```

Comment:
what if we do something like:

```
OPENWISP_CELERY_COMMAND_FLAGS="--concurrency 1"
```

So that we can configure any flag more easily and avoid all those ifs below which would be quite painful to maintain?

This solution also allow for great flexibility in configuring the celery workers.